addons/l10n_dk_nemhandel/views/account_journal_dashboard_views.xml

```diff
@@ -6,27 +6,6 @@
         <field name="inherit_id" ref="account.account_journal_dashboard_kanban_view" />
         <field name="arch" type="xml">
             <data>
-                <xpath expr="//kanban" position="inside">
```

Comment:
I keep the empty view as it would need an upgrade script.
I will remove it in master